style.css

```diff
@@ -0,0 +1,849 @@
+		height: 35%;
+		min-width: 298px;
+		width: 72%;
+	}
+
+	.click-text {
+		display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 56%;
+	}
+
+	.card-1, .card-2, .card-3 {
+		width: 35%;
+	}
+
+	.QA-section {
+		margin: 90px 30px 20px 30px;
+	}
+
+}
+
+/** Tablet landscape up **/
+@media (min-width: 900px) {
+	h1 {
+		text-align: start;
+	}
+
+	h5 {
+		text-align: start;
+		padding: 0;
+	}
+	
+	.topnav {
+		display: flex;
+		flex-direction: row;
+		width: auto;
+		height: 90px;
+		justify-content: space-between;
+		align-items: center;
+		padding: 0 65px;
+	}
+	
+	.topnav #myLinks {
+		display: flex;
+		flex-direction: row;
+		height: fit-content;
+		align-items: center;
+		background-color: inherit;
+	}
+	
+	.topnav a {
+		color: black;
+		font-size: 10px;
+	}
+	
+	.topnav a:hover {
+		color: hsl(0, 94%, 66%);
+	}
+	
+	.topnav a.icon {
+		display: none;
+	}
+	
+	.hamburger, .close, .social-media {
+		display: none;
+	}
+	
+	.log-in {
+		background-color: hsl(0, 94%, 66%);
+		font-size: 10px;
+		height: 30px;
+	}
+	
+	.log-in:hover, .red-btn:hover {
+		background-color: white;
+		border: 1.5px solid hsl(0, 94%, 66%);
+		color: hsl(0, 94%, 66%);
+		border-radius: 5px;
+	}
+	
+	.active {
+		display: none;
+	}
+
+	main {
+		flex-direction: row;
+		margin-top: 0;
+	}
+
+	.background-shape {
+		width: 345px;
+		min-height: unset;
+		height: 225px;
+  	border-top-left-radius: 117px;
+		border-bottom-left-radius: 117px;
+		top: 269px;
+	}
+
+	.hero-img {
+		width: 396px;
+		right: 9px;
+		top: 193px;
+	}
+
+
+	.text-container {
+		display: flex;
+ 	 flex-direction: column;
+		width: 300px;
+		margin-left: 87px;
+		margin-top: 0;
+	}
+
+	.btn-container button {
+		width: 134px;
+	}
+}
+
+
+/***** Desktop *****/
+/* @media (min-width: 1440px){ */
+@media (min-width: 1024px){
+	h1 {
+		margin-bottom: 0;
+		font-size: 2.7vw;
+		margin-top: 100px;
+	}	
+
+	h2 {
+		font-size: 1.9em;
+	}
+	h5 {
+		font-size: 1.3em;
+	}	
+
+	.topnav {
+		padding: 0 150px;
+	}
+
+	.topnav .social-media {
+		display: none;
+	}
+
+	.hero-img {
+		min-width: unset;
+		width: 408px;
+	 	right: 82px;
+		top: 193px;
+	}
+
+	.background-shape {
+		height: 250px;
+		top: 273px;
+	}
+
+	.text-container {
+		max-width: 492px;
+		width: 37%;
+		margin-left: 150px;
+		margin-top: 0;
+	}
+
+	.btn-container button {
+		width: 150px;
+	}
+
+	.btn-container .blue-btn:hover, .card-section .blue-btn:hover {
+		background-color: white;
+		border: 1.5px solid hsl(231, 69%, 60%);
+		border-radius: 5px;
+		color: hsl(231, 69%, 60%);
+	}
+
+	.btn-container .gray-btn:hover {
+		background-color: white;
+		border: 1.5px solid gray;
+		border-radius: 5px;
+		color: black;
+	}
+
+	.features {
+		display: flex;
+  	flex-direction: column;
+  	margin-top: 190px;
+  	width: 600px;
+  	margin-right: 20%; /*having problems with this margin*/
+  	margin-left: 20%;
+	}
+	.ul-div {
+		display: flex;
+		flex-direction: row;
+		width: 700px;
+		border-bottom: 0.8px solid gray;
+	}
```

Comment:
I think there are several ways to style this but one suggestion is to remove 
the border (i.e. border: none;) and use box-shadow (or something similar).